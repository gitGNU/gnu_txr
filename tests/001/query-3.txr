@#
@# This file is in the public domain.
@# It was authored by Kaz Kylheku <kkylheku@gmail.com> in 2009
@#
@/.*/@# skip header
@(collect)
@  (coll)@{TOKEN /[^ ]+/}@(end)
@  (bind (UID PID PPID C STIME TTY TIME . CMD) TOKEN)
@  (cat CMD) @#
@  (forget TOKEN)
@(end)
@(output)
@(repeat)
@{UID 8} @{PID -5}
@(first)
UID        PID
@{UID 8} @{PID -5}
@(empty)
No processes!
@(end)
@(end)
