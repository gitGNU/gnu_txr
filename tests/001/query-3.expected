UID        PID
root         1
root         2
root         3
root         4
root         5
root        16
root        29
root        17
root        28
root       103
root       175
root       186
root       870
root      1068
root      1235
root      1236
root      1620
root      1624
rpc       1645
rpcuser   1665
root      1698
root      1766
root      1790
root      1791
root      1821
root      1839
root      1851
root      1887
root      1902
root      1921
root      1925
root      1926
root      1927
root      1928
root      1929
root      1930
root      1931
root      1932
root      1936
root      1963
xfs       1989
daemon    2008
dbus      2027
root      2041
root      2052
root      2062
root      2124
root      2184
root      2354
kaz       2551
kaz       2579
kaz       2625
kaz       2626
kaz       2631
kaz       2634
kaz       2636
kaz       2638
kaz       2644
kaz       2661
kaz       2685
kaz       2689
kaz       2691
kaz       2693
kaz       2695
kaz       2698
kaz       2701
kaz       2707
root      2717
kaz       2718
kaz       2720
kaz       2722
kaz       2726
kaz       2728
root     30737
root     31905
kaz      31907
kaz      31908
root     32672
kaz      32674
kaz      32675
root     27121
root     27243
root     27682
kaz      27684
kaz      27685
kaz       6481
